lib/cmd/clean.cc

```diff
@@ -0,0 +1,48 @@
+// std
+#include <iostream>
+#include <string>
+
+// external
+#include <spdlog/spdlog.h> // NOLINT(build/include_order)
+
+// internal
+#include "poac/cmd/clean.hpp"
+#include "poac/config.hpp"
+#include "poac/data/manifest.hpp"
+#include "poac/util/validator.hpp"
+
+namespace poac::cmd::clean {
+
+[[nodiscard]] Result<void>
+clean(const Options& opts) {
+  const auto profile =
+      Try(util::validator::valid_profile(opts.profile, opts.release)
+              .map_err(to_anyhow));
+  if (profile.has_value() && profile.value() != "debug"
+      && profile.value() != "release") {
+    return Err<build::UnsupportedProfile>(profile.value());
+  }
+
+  const auto path = profile.has_value()
+                        ? config::path::output_dir / profile.value()
+                        : config::path::output_dir;
+
+  spdlog::trace("Removing ./{}", path);
+
+  if (std::filesystem::exists(path)) {
+    log::status("Removing", std::filesystem::canonical(path).string());
+    std::filesystem::remove_all(path);
```

Comment:
We have a shorthand for `std::filesystem`.

https://github.com/poacpm/poac/blob/11d2a27c83aacde88d573e3307fd7cd2959c429a/include/poac/poac.hpp#L60

Reply:
✔️